keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -264,6 +268,31 @@ state, it will either trigger [Un-reserve](#un-reserve) plugins (on failure) or
 
 *Note: This concept used to be referred to as "assume".*
 
+### Pre-Preempt
```

Comment:
Do we have use cases for this and Post-Preempt? Otherwise, I would say, we can start simple by just having one extension point.

Reply:
No actually.  I also agree with starting from the simplest. I'll delete this extension point.